core/sdk-core/src/main/java/software/amazon/awssdk/core/retry/backoff/FixedDelayBackoffStrategy.java

```diff
@@ -24,6 +24,9 @@
 
 /**
  * Simple backoff strategy that always uses a fixed delay for the delay before the next retry attempt.
+ *
+ * @deprecated Use instead {@link software.amazon.awssdk.retries.api.BackoffStrategy} and
+ * {@link software.amazon.awssdk.retries.api.BackoffStrategy#fixedDelay(Duration)}.
  */
 @SdkPublicApi
```

Comment:
Why dont we habe @Depreacted annotation for this ?

Reply:
I missed this one, I will add it, good catch, thanks!